using System;

namespace Exception
{
    internal class Program
    {
        static void Main(string[] args)
        {
            Exception exception = null;
            try
            {

            }
            catch (Exception ex)
            {

            }
        }
    }
}
